yarn.lock

```diff
@@ -12771,16 +12737,6 @@ ultron@1.0.x:
   resolved "https://registry.yarnpkg.com/ultron/-/ultron-1.0.2.tgz#ace116ab557cd197386a4e88f4685378c8b2e4fa"
   integrity sha1-rOEWq1V80Zc4ak6I9GhTeMiy5Po=
 
-underscore.string@~2.4.0:
-  version "2.4.0"
-  resolved "https://registry.yarnpkg.com/underscore.string/-/underscore.string-2.4.0.tgz#8cdd8fbac4e2d2ea1e7e2e8097c42f442280f85b"
-  integrity sha1-jN2PusTi0uoefi6Al8QvRCKA+Fs=
```

Comment:
There was a chain of dependencies yielding to this module.
We couldn't upgrade just this module because of the requested dependencies (they were "~" dependencies, so this couldn't be upgraded to the next major version, which was needed here).

I found which package was transitively depending to this by running `yarn why underscore.string` in the command line, then tried to upgrade that package `flow-coverate-report`, and it worked just fine.

I ran `yarn flow-coverage-report` in the command line after the upgrade and it worked fine.